src/main/java/compal/logic/parser/ParserManager.java

```diff
@@ -0,0 +1,202 @@
+                    break;
+                case CMD_LIST:
+                    ListCommand list = new ListCommand(compal);
+                    list.parseCommand(cmd);
+                    break;
+                case CMD_CLEAR:
+                    ClearCommand clear = new ClearCommand(compal);
+                    clear.parseCommand(cmd);
+                    break;
+                case CMD_DONE:
+                    DoneCommand done = new DoneCommand(compal);
+                    done.parseCommand(userInput);
+                    break;
+                case CMD_DELETE:
+                    DeleteCommand delete = new DeleteCommand(compal);
+                    delete.parseCommand(userInput);
+                    break;
+                case CMD_EVENT:
+                    EventCommand event = new EventCommand(compal);
+                    event.parseCommand(userInput);
+                    break;
+                case CMD_DEADLINE:
+                    DeadlineCommand deadline = new DeadlineCommand(compal);
+                    deadline.parseCommand(userInput);
+                    break;
+                case CMD_LECT:
+                case CMD_TUT:
+                case CMD_SECT:
+                case CMD_LAB:
+                    AcadCommand acad = new AcadCommand(compal);
+                    acad.parseCommand(userInput);
+                    break;
+                case CMD_RECUR_TASK:
+                    RecurTaskCommand recurTask = new RecurTaskCommand(compal);
+                    recurTask.parseCommand(userInput);
+                    break;
+                case CMD_FIND:
+                    FindCommand findCommand = new FindCommand(compal);
+                    findCommand.parseCommand(userInput);
+                    break;
+                case CMD_VIEW:
+                    ViewCommand viewCommand = new ViewCommand(compal);
+                    viewCommand.parseCommand(userInput);
+                    break;
+                case CMD_SET_REMINDER:
+                    SetReminderCommand setReminderCommand = new SetReminderCommand(compal);
+                    setReminderCommand.parseCommand(userInput);
+                    break;
+                case CMD_HELP:
+                    HelpCommand helpCommand = new HelpCommand(compal);
+                    helpCommand.parseCommand(userInput);
+                    break;
+                case CMD_FIND_FREE_SLOT:
+                    FindFreeSlotCommand findFreeSlotCommand = new FindFreeSlotCommand(compal);
+                    findFreeSlotCommand.parseCommand(userInput);
+                    break;
+                case CMD_EDIT:
+                    EditCommand ec = new EditCommand(compal);
+                    ec.parseCommand(userInput);
+                    break;
+
+                default:
+                    compal.ui.printg(MESSAGE_INVALID_COMMAND);
+                    throw new Compal.DukeException(MESSAGE_INVALID_COMMAND);
+                }
+            }
```

Comment:
Indentation issue on switch statements. Perhaps an if else here would be easier to read.